lib/routes.js

```diff
@@ -151,6 +151,14 @@ module.exports = async function (app, opts) {
     app.route(getOptions)
   }
 
+  app.addContentTypeParser('application/graphql', function (req, done) {
+    var data = ''
+    req.on('data', chunk => { data += chunk })
+    req.on('end', _ => {
+      done(null, { query: data })
```

Comment:
How are variables being sent?

Reply:
Variables are not supported with application/graphql content-type, just query. See https://graphql.org/learn/serving-over-http/#post-request second special case.

curl -H "Content-Type:application/graphql" -XPOST -d "{ add(x: 2, y: 1) }" http://localhost:3000/graphql

Reply:
Can you add a bit of docs in https://github.com/mcollina/fastify-gql#post-graphql describing this?

Reply:
I have updated documentation and added new test with mutation via POST and application/graphql.